test/unit/test_NFTestAssert.py

```diff
@@ -101,3 +101,26 @@ def test_assert_expected(mock_assert):
     mock_assert.return_value.get_assert_method = lambda: lambda x, y: False
     with pytest.raises(AssertionError):
         NFTestAssert.assert_expected(mock_assert())
+
+@pytest.mark.parametrize(
+    'glob_return_value,case_pass',
+    [
+        ([], False),
+        (['a', 'b'], False),
+        (['a'], True)
+    ]
+)
+@mock.patch('glob.glob')
+@mock.patch('nftest.NFTestAssert.NFTestAssert', wraps=NFTestAssert)
+def test_identify_assertions_files(mock_assert, mock_glob, glob_return_value, case_pass):
+    ''' Tests for proper file identification '''
+    mock_assert.return_value.actual = ''
+    mock_assert.return_value.expect = ''
+
+    mock_glob.return_value = glob_return_value
+
+    if case_pass:
+        NFTestAssert.identify_assertion_files(mock_assert())
```

Comment:
This test _might_ add clarity if the `NFTestAssert` class is instantiated and called normally. It takes a bit of mental overhead to recognize exactly what's happening here. If a test is created for `resolve_single_path`, then it likely makes more sense not to test the `NFTestAssert.identify_assertion_files` method like this.

Reply:
To fully instantiate it, the class would also instantiate NFTestEnv and a logger, do you think it would make sense to mock those for instantiation or actually fully go through with it?

Reply:
Got it, I updated that test to instantiate the actual class with mocked dependencies